### PR TITLE
EVM stuff

### LINKED ISSUE #14672: Take BNB out of experimental features

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- remove Polygon backwards compatibility with MATIC Blockbooks
- put BNB Smart Chain out of experimental features
- add Optimism into experimental features (a lot of changes needed before 24.11 freeze)
  - wrong symbol
  - too big fee
  - works only with VPN **!!!CRITICAL!!!**
- remove coin icons from components, I forgot to remove it here https://github.com/trezor/trezor-suite/pull/14463

## Resolve

closes #14672

## Screenshots:
Screenshot 2024-10-04 at 20 53 09
Screenshot 2024-10-04 at 20 54 24
Screenshot 2024-10-04 at 20 49 30

### CODE REVIEW

File: suite-common/icons-deprecated/src/icons.ts [hunk 1]
```diff
@@ -238,6 +238,7 @@ export const cryptoIcons = {
     dsol: require('../assets/cryptoIcons/dsol.svg'),
     etc: require('../assets/cryptoIcons/etc.svg'),
     eth: require('../assets/cryptoIcons/eth.svg'),
+    op: require('../assets/cryptoIcons/op.svg'),
```
Comment (jvaclavik) [inline suite-common/icons-deprecated/src/icons.ts:241]: do we need duplicity here?

**Review by jvaclavik — APPROVED** [review]
Looks good except duplicity of icons

File: suite-common/icons-deprecated/src/icons.ts (hunk 1 above)
Comment (tomasklim) [reply suite-common/icons-deprecated/src/icons.ts:241]: There is `CryptoIcon` component used in react native which is using this. So unfortunately, for now it has to be deprecated

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: web 24.11.0 (0f49b0fadd15e07a61d4fb88d47a4f7e2d5158db)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.4 regular (revision d71d9e9c343ad04a7f28d6b02f8d3f8945af30d6)
 - Transport: WebUsbTransport

**Pull request merged**